lib/Data/Chronicle/Reader.pm

```diff
@@ -199,7 +199,7 @@ sub get_for_period {
     for my $id_value (keys %$db_data) {
         my $db_value = $db_data->{$id_value}->{value};
 
-        push @result, JSON::from_json($db_value);
+        push @result, JSON::MaybeXS->new->decode($db_value);
```

Comment:
This is presuming a Unicode character string value out of the `dbic` instance. That just gets passed in to its constructor, so I can't easily see if that's correct here.